.pylintrc

```diff
@@ -81,7 +81,8 @@ disable=missing-docstring,
       missing-module-docstring, # temp-pylint-upgrade
       import-error, # needed as a workaround as reported here: https://github.com/open-telemetry/opentelemetry-python-contrib/issues/290
       cyclic-import,
-      not-context-manager
+      not-context-manager,
+      broad-exception-raised
```

Comment:
this should not be added then

Reply:
Did that and `lint` [failed](https://github.com/open-telemetry/opentelemetry-python-contrib/pull/2471/commits/4532a7f6ceb1d03927771fbdceb0a8bab157b158).

Reply:
@ocelotl I've opened a draft PR to fix them, failing because I got some the pylint exception wrong and missed to update tests when used different Exception types, will update it tomorrow